src/app/@ansyn/imagery/model/base-imagery-map.ts

```diff
@@ -40,6 +40,10 @@ export abstract class BaseImageryMap<T = any> {
 
 	abstract getCenter(): Observable<Point>;
 
+	public getTargetElement(): HTMLElement {
```

Comment:
why not an abstract method?

Reply:
This way you don't need to implement it evetywhere.
Actually... I will remove it. don't need it anymore.